packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -443,11 +443,7 @@ export const TracksTable = ({
             includeShare: !isUnlisted || isOwner,
             includeEmbed: !isUnlisted,
             includeEdit: !disabledTrackEdit,
-            includeAddToPlaylist:
-              !isUnlisted &&
-              (!isStreamGated ||
-                (isContentUSDCPurchaseGated(streamConditions) &&
-                  hasStreamAccess)),
+            includeAddToPlaylist: !isUnlisted || isOwner,
```

Comment:
there must be other places we need to make this change, right?

Reply:
dang u know wat..